.github/actions/common-versioning/action.yml

```
@@ -78,18 +89,28 @@ runs:
-          } >> "$GITHUB_OUTPUT"
+            echo "version=${UPDATED_VERSION}"
+            echo "full_version=${FULL_VERSION}"
+            echo "short_sha=${SHORT_SHA}"
+          } >> $GITHUB_OUTPUT
           
-          echo "Debug: Contents of GITHUB_OUTPUT:"
+          # Verify outputs were set
+          echo "Debug: Verifying outputs were set correctly:"
+          echo "GITHUB_OUTPUT contents:"
           cat "$GITHUB_OUTPUT"
           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)